### PR TITLE
replace activity links with updated ones

### PR DESCRIPTION
**Pull request by hannahkane**
This addresses the link changes specified in https://github.com/mozilla/learning-networks/issues/357

@mmmavis - can you review if I did this correctly, please? Thank you!

### CODE REVIEW

File: pages/web-lit-basics-two.jsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ var curriculumList = [
         caption: "CC0 1.0 by Tungalik",
         subtitle: "Understanding Community Participation",
         description: "Learners will create and research survey questions about their community's Web use, learning about <strong>collaborating, community participation, and open practices</strong>.",
-        link: "https://chadsansing.github.io/curriculum-testing/web-lit-basics-two/session01-why-do-we-use-the-web.html"
+        link: "http://chadsansing.github.io/curriculum-testing/curriculum-migration/web-lit-basics-two-migrated/session01-why-do-we-use-the-web.html"
```
Comment (mmmavis) [inline pages/web-lit-basics-two.jsx]: Hey @hannahkane can you update the links so they are `https` instead of `http` :wink:

**Comment by mmmavis** [comment]
Wow your first PR @hannahkane ! :+1: 

It's better to use `https` links whenever we can so @hannahkane could you please update this PR again?

1. `git status` to check if there's any local uncommitted changes. (if yes IRC me, if no, go to Step 2)
2. `git checkout issue-357-update-weblitbasicsII` to switch to this branch
3. update links in `pages/web-lit-basics-two.jsx`
4. git commit and git push as usual :smile: 

Ping me anytime when you need help!

**Comment by hannahkane** [comment]
thanks, @mmmavis. I think this one is ready for review again.

**Comment by mmmavis** [comment]
:+1: looking great! Congratz on making your PR @hannahkane  :dancers: :dancers: :dancers:

**Pull request merged**